extensions/adobe/experience/campaign/mutationevent.schema.json

```diff
@@ -19,17 +19,12 @@
           "format": "date-time",
           "description": "The time when the first event of the interaction occurred."
         },
-        "xdm:web": {
-          "title": "Web",
-          "$ref": "https://ns.adobe.com/xdm/context/webinfo",
-          "description": "The information related to web page and link of the ExperienceEvent."
-        },
         "xdm:directMarketing": {
           "title": "Direct Marketing",
           "$ref": "https://ns.adobe.com/xdm/context/direct-marketing",
           "description": "The events and properties related to direct/outbound marketing such as email, direct mail, texts and in-app notifications."
         },
-        "https://ns.adobe.com/experience/campaign/message": {
```

Comment:
As far as I know, this should not change the XDM path to which we currently send our data, is this understanding correct?

Reply:
correct this should not change the path.